freedomofpress/securedrop-client
# use dialog fixtures and fix flaky tests

# Description

Fixes https://github.com/freedomofpress/securedrop-client/issues/1139

# Test Plan

Confirm https://github.com/freedomofpress/securedrop-client/issues/1139 is fixed.

- run script from STR section of https://github.com/freedomofpress/securedrop-client/issues/1139 (takes approx 17 minutes if there are no failures: 90-120 seconds each test run) 
- [ ] confirm no dialog errors
- modify script to execute `make test` instead of `make test-random` and modify how many times you loop if you're not satified with 10 (I've tested with 100)
- run script  (takes approx 5 minutes for 10 loops and if there are no failures) 
- [ ] confirm no dialog error
